apps/webapp/web/static/js/docs/DocContent.js

```diff
@@ -0,0 +1,24 @@
+// file: docs/DocContent.js
```

Comment:
What about naming this component/file DocPage since basically you're going to render a .md page inside it? what do you think @edipox ?